actions/action_test.go

```diff
@@ -281,87 +281,3 @@ func TestMatchedActions(t *testing.T) {
 		})
 	}
 }
-
-func TestMatchActions(t *testing.T) {
```

Comment:
Why delete?

Reply:
because I turned it into private as you asked... return it as public